images/miq-app/docker-assets/appliance-initialize.sh

```diff
@@ -24,13 +24,13 @@ restore_pv_data
 cd ${APP_ROOT}
 bin/rake evm:deployment_status
 
+# Generate httpd certificate
+/usr/bin/generate_miq_server_cert.sh
```

Comment:
I think it should, but it wasn't as easy as I originally thought it would be.  So, I'd like to do that in a follow-up PR